conjur/wrapper/http_wrapper.py

```diff
@@ -67,36 +77,40 @@ def invoke_endpoint(http_verb: HttpVerb, endpoint: ConjurEndpoint, params: dict,
     # server pem received during initialization of the client
     # pylint: disable=not-callable
 
+    loop = asyncio.get_event_loop()
     try:
-        response = invoke_request(http_verb,
-                                  url, *args,
-                                  query=query,
-                                  ssl_verify=True,
-                                  auth=auth,
-                                  headers=headers)
+        response = loop.run_until_complete(invoke_request(http_verb,
```

Comment:
I feel this is out-of-scope for me, and since we run on python 3.6, I can't use `asyncio.run`.
I don't like very much the proposed code, as there are 2 options, both seem very messy.

Reply:
BTW we use python 3.9 (and will upgrade to 3.10) wrapping the CLI. so maybe it will be best for you to update your python.